src/ApiEndpoints.NSwag/Ardalis.ApiEndpoints.NSwag.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
There's already a https://github.com/ardalis/ApiEndpoints/blob/main/src/Ardalis.ApiEndpoints.NSwag/Ardalis.ApiEndpoints.NSwag.csproj why is this project file needed? 
If you want to rename `Ardalis.ApiEndpoints.NSwag` folder to `ApiEndpoints.NSwag`, you need to also delete files from `Ardalis.ApiEndpoints.NSwag`